src/storage/txn/flow_controller/tablet_flow_controller.rs

```diff
@@ -191,6 +218,10 @@ impl FlowInfoDispatcher {
                                         ),
                                         discard_ratio,
                                     ));
+                                    info!(
+                                        "add FlowChecker region_id {}. Total FlowChecker count {}",
+                                        region_id, current_count
+                                    );
```

Comment:
```suggestion
                                    info!(
                                        "add FlowChecker";
                                        "region_id" => region_id,
                                        "current_count" => current_count,
                                    );
```

It might be verbose though.